deploy_exchange/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to Anypoint Exchange
+description: Deploy to Anypoint Exchange with Maven
+inputs:
+  connected_app_client_id:
+    description: CloudHub connected app client ID
+    required: true
+  connected_app_client_secret:
+    description: CloudHub connected app client secret
+    required: true
+  maven_settings_path:
+    description: Maven settings file path
+    required: true
+  mule_file_path:
+    description: Mule file path
+    required: true
+  cloudhub_business_group_id:
+    description: CloudHub business group ID
```

Comment:
Should this have `required: true` as well?

Reply:
true, this should be required @toby-thanathip - I will modify this

Reply:
added in https://github.com/nimblehq/mulesoft-actions/pull/12/commits/01b8f21fe128de73e3e6514fcff96cebc1d9adec @toby-thanathip